tutorials/getting_started/configuration.md

```diff
@@ -0,0 +1,344 @@
+specified by our configuration file.
+
+## `dataset_reader`
+
+The first section of our configuration file defines the `dataset_reader`:
+
+```js
+  "dataset_reader": {
+    "type": "sequence_tagging",
+    "word_tag_delimiter": "/",
+    "token_indexers": {
+      "tokens": {
+        "type": "single_id",
+        "lowercase_tokens": true
+      },
+      "token_characters": {
+        "type": "characters"
+      }
+    }
+  },
+
+```
+
+Here we've specified that we want to use the `DatasetReader` subclass that's registered
+under the name `"sequence_tagging"`. Unsurprisingly, this is the
+[`SequenceTaggingDatasetReader`](http://docs.allennlp.org/en/latest/api/allennlp.data.dataset_readers.html#allennlp.data.dataset_readers.sequence_tagging.SequenceTaggingDatasetReader)
+subclass. This reader assumes a text file of newline-separated sentences, where each sentence looks like
+
+```
+word1{wtd}tag1{td}word2{wtd}tag2{td}...{td}wordn{wtd}tagn
+```
+
+For some "word tag delimiter" `{wtd}` and some "token delimiter" `{td}`.
+
+_Our_ data files look like
+
+```
+The/at detectives/nns placed/vbd Barco/np under/in arrest/nn
+```
+
+which is why we need to specify
+
+```js
+    "word_tag_delimiter": "/",
+```
+
+We don't need to specify anything for the "token delimiter",
+since the default split-on-whitespace behavior is already correct.
+
+If you look at the code for `SequenceTaggingDatasetReader.read()`,
+it turns each sentence into a `TextField`
+of tokens and a `SequenceLabelField` of tags. The latter isn't
+really configurable, but the former wants a dictionary of
+[TokenIndexer](http://docs.allennlp.org/en/latest/api/allennlp.data.token_indexers.html#allennlp.data.token_indexers.token_indexer.TokenIndexer)s
+that indicate how to convert the tokens into arrays.
+
+Our configuration specifies two token indexers:
+
+```js
+    "token_indexers": {
+      "tokens": {
+        "type": "single_id",
+        "lowercase_tokens": true
+      },
+      "token_characters": {
+        "type": "characters"
+      }
+    }
+```
+
+The first, `"tokens"`, is a
+[`SingleIdTokenIndexer`](http://docs.allennlp.org/en/latest/api/allennlp.data.token_indexers.html#allennlp.data.token_indexers.single_id_token_indexer.SingleIdTokenIndexer)
+that just represents each token (word) as a single integer.
+The configuration also specifies that we *lowercase* the tokens before encoding;
+that is, that this token indexer should ignore case.
+
+The second, `"token_characters"`, is a
+[`TokenCharactersIndexer`](http://docs.allennlp.org/en/latest/api/allennlp.data.token_indexers.html#allennlp.data.token_indexers.token_characters_indexer.TokenCharactersIndexer)
+that represents each token as a list of int-encoded characters.
+
+Notice that this gives us two different encodings for each token.
+Each encoding has a "namespace", in this case `"tokens"` and `"token_characters"`.
+The `SequenceLabelField` also has a namespace, `"labels"`.
+
+## non-padded namespaces
```

Comment:
I was hoping this would be a topic almost all users could ignore, that just shows up in some deep dive.  It's unfortunate that we need to use it here.  Can you construct the validation dataset such that it's not necessary in this tutorial?

Reply:
yeah, that's a good idea, I'll just filter out the lines with the stray tags

Reply:
ok, I fixed the datasets and removed all references to non-padded namespaces from the tutorial and config file

Reply:
:+1: